_includes/docs/user-guide/firmware.md

```diff
@@ -42,10 +58,84 @@ Deletion of the firmware packages that is assigned to at least one device or dev
 
 ## Update process
 
+Attributes show information about the new firmware update.
+
+Where
+- fw_checksum - attribute that responsible for verifying if the received file is the same as the sent one.
+- fw_checksum_algorithm - the algorithm by which the checksum is calculated. 
+- fw_size - size of the firmware file in bytes.
+- fw_title - name of the firmware.
+- fw_version - version of the firmware.
+- fw_state - [status](/docs/{{docsPrefix}}user-guide/firmware/?remoteintegrationdockerinstall=mqtt#dashboard) of the updating process. 
 
+{% include images-gallery.html imageCollection="fw-attributes" %}
+
+When we receive an attribute update, we download the firmware. According to them, we determine if the firmware has been updated and whether it suits us.
+
+{% capture contenttogglespec %}
+HTTP<br/>%,%http%,%templates/install/http-firmware.md%br%
+MQTT<br/>%,%mqtt%,%templates/install/mqtt-firmware.md%br%
+CoAP<br/>%,%aws%,%templates/install/coap-firmware.md%br%{% endcapture %}
+{% include content-toggle.html content-toggle-id="remoteintegrationdockerinstall" toggle-spec=contenttogglespec %}
 
 ## Dashboard
 
+ThingsBoard provides the summary of the firmware update to monitor and track the firmware update status of your device, such as which devices are updating right now, 
+any boot issues, and which ones have already been updated.
+It is created automatically when you add firmware to ThingsBoard.
```

Comment:
It is created automatically for each new tenant you add to ThingsBoard. You can also download the dashboard json here (link to the file on github)

Reply:
all issues fixed